lib/certs.js

```diff
@@ -87,14 +87,16 @@ function detect(options, callback) {
 							commonName = cert.subject.getField('CN').value;
 
 							if (!validOnly || !invalid) {
+								const teamId = cert.subject.attributes.find(attr => attr.name === 'organizationalUnitName')
 								dest.push({
 									name: appc.encoding.decodeOctalUTF8(name ? commonName.substring(name.length) : commonName).trim(),
 									fullname: appc.encoding.decodeOctalUTF8(commonName).trim(),
 									pem: pem,
 									before: cert.validity.notBefore,
 									after: cert.validity.notAfter,
 									expired: expired,
-									invalid: invalid
+									invalid: invalid,
+									teamId: teamId ? teamId.value : undefined
```

Comment:
You can just do `teamId && teamId.value`. If `teamId` is `undefined`, then the result will be `undefined`.